products/compute/inspec.yaml

```diff
@@ -13,9 +13,6 @@
 
 --- !ruby/object:Provider::Inspec::Config
 manifest: !ruby/object:Provider::Inspec::Manifest
-  version: '0.1.0'
-  source: 'FIXME'
```

Comment:
Could we figure this out? Seems like neither of these 'FIXME' properties are necessary.

Reply:
That's why they have been removed